### DISCUSSION TITLE
Unexpected response code: 403

### DISCUSSION THREAD

**Discussion by Tivoyagefeak** [mozilla/hubs#3419] [Hubs Cloud Questions] [answered]
Hello everyone. 
I have a big problem with my Hubs Cloud Installation. Its hosted via the AWS Multiserver solution and I created a custom client. After adding an etry to the CORS Settings all of the sudden im no longer able to enter the admin panel or open a room. The network console spams phoenix.js:844 WebSocket connection to 'wss://HubsWebsite/socket/websocket?vsn=2.0.0' failed: Error during WebSocket handshake: Unexpected response code: 403. I already tried to get rid of the custom client but the error persists. I also killed the server instances with the EC2 manager twice with no success. Any ideas?

Chrome Console:
![image](https://user-images.githubusercontent.com/5426151/99981678-f9498500-2da9-11eb-8fd8-e964c974bbcc.png)

**Comment by Tivoyagefeak** [comment, marked as answer]
After some more diggin I found out how to reset CORS settings with AWS. #2890 gives a detail information about how to do that. I first couldnt be sure if it was because of the CORS settings, therefore I used the wrong searching parameters I guess

**Reply by robinkwilson** [reply]
Marking as answer for now, let me know if this doesnt fix your problem!
